apps/ceos/src/assets/header/menuBtn.tsx

```diff
@@ -1,13 +1,24 @@
 import { KeyOfPalette, theme } from '../../../../../packages/ui';
```

Comment:
이부분 절대경로로 수정하는게 더 좋지 않을까용
```suggestion
import { KeyOfPalette, theme } from '@ceos-fe/ui';
```